proc/threads.go

```diff
@@ -323,3 +324,26 @@ func (thread *Thread) Scope() (*EvalScope, error) {
 	}
 	return locations[0].Scope(thread), nil
 }
+
+func (thread *Thread) SetCurrentBreakpoint(excluded *Breakpoint) error {
```

Comment:
That's why that parameter existed. However I realized that I can just singlestep all the threads stopped on a breakpoint and then call resume on all threads indiscriminately, no need check for hitting a breakpoint after the singlestepping phase, it will never happen anyway. I have changed the patch to do this.